### DISCUSSION TITLE
support for changelist a.k.a changes: g, g;

### DISCUSSION THREAD

**Discussion by IamMashed** [JetBrains/ideavim#869] [Q&A] [answered]
Hello Guys, 
does anyone work on achieving g, and g; commands in ideavim? 

those are very useful and work in vscode already.

**Comment by VasoTodorovic** [comment]
You have built in actions in Intellij Last Edited Location, and Previous Edited Location. I mapped that to g; and g,
It has difference then vim. In Intellij will return you to previous/next edited location in whole project in vim it will return you to last edited in current buffer. 
I even tried to have both option in VScode, to return last edited in buffer and in whole project. But it will have some confusing jumps sometime. In Intellij as I now you can just to jump to last edited and previous in whole project.

**Reply by IamMashed** [reply]
how do you map g; or g, in intellij? 
i tried various combinations, none work for me. 
![Screenshot 2024-05-13 at 12 39 09 PM](https://github.com/JetBrains/ideavim/assets/10993575/01d0c9d4-e3f6-420f-aabd-44e583c5cbae)

**Reply by VasoTodorovic** [reply, marked as answer]
```
map g; :action JumpToLastChange<Enter>
map g, :action JumpToNextChange<Enter>
```

try this one and tell if it is working, for me it's doing its job

**Reply by VasoTodorovic** [reply]
I just notice now do you config via .ideavimrc or you are trying to config via intellij GUI because idavim is config with .idavimrc file, it is similar to vimrc or .lua file for vim/neovim
